docs/sphinx/install.rst

```diff
@@ -140,15 +195,255 @@ To build the CUDA Quantum Python API from source using pip, run the following co
     cd cuda-quantum && ./scripts/install_prerequisites.sh
     pip install .
 
-For more information about building the entire C++ and Python API's, please see `Building from Source`_.
+For more information about building the entire C++ and Python API from source, we refer to the `CUDA Quantum GitHub repository`_.
 
-Building from Source
-------------------------------
+.. _local-development-with-vscode:
 
-Instructions for building the Python wheels from source are given in the section :ref:`install-python-wheels`.
-For more information about building the entire C++ and Python API from source, we refer to the `CUDA Quantum GitHub repository`_.
+Development with VS Code
+------------------------------------
 
-.. _CUDA Quantum GitHub repository: https://github.com/NVIDIA/cuda-quantum/blob/main/Building.md
+To facilitate application development with, for example, debugging, code completion, hover information, and so on, 
+we recommend using `VS Code <https://code.visualstudio.com/>`_. VS Code provides a seamless
+development experience on all platforms, and is also available without installation via web browser. 
+This sections describes how to connect VS Code to a running container on your machine.
+The section on :ref:`connecting to a remote host <connect-to-remote>` contains information on
+how to set up your development environment when accessing CUDA Quantum on a remote host instead.
+
+.. _docker-in-vscode:
+
+Using a Docker container
+++++++++++++++++++++++++++++++++++++++++
+
+Before connecting VS Code, open a terminal/shell, 
+and start the CUDA Quantum Docker container following the 
+instructions in the :ref:`section above <install-docker-image>`. 
+
+If you have a local installation of `VS Code <https://code.visualstudio.com/>`_ 
+you can connect to the running container using the  
+`Dev Containers <https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers>`__ extension. If you want to use VS Code in the web browser, please follow the instructions
+in the section `Developing with Remote Tunnels`_ instead.
+
+After installing the
+`Dev Containers <https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers>`__ extension, launch VS Code, open the Command Palette with `Ctrl+Shift+P`, and enter 
+.. spellcheck-disable
```

Comment:
This doesn't look right in the docs preview. It is showing the `spellcheck-disable` and `spellcheck-enable` strings.

Reply:
Good catch! I forgot to search for that in the published docs...
Let me give it a try to hack it up differently - looks like comments in rst are only recognized if they surrounded by empty lines it seems (or this is a bug in sphinx), so I'll abuse a replacement instead.